src/App.vue

```diff
@@ -358,6 +359,18 @@ export default class App extends Mixins(
   }
 }
 
+.namerequest-sbc-breadcrumb {
+  .container {
+    max-width: 1360px; // should match auth-web, etc
```

Comment:
Should this be in the shared component?

Reply:
This is just to align it with the edge of the SBC Auth header. If the breadcrumb component will only ever be used underneath the sbc auth header then it can most likely be moved to the shared component.

Reply:
It looks to me like the Breadcrumb component looks OK in Storybook:
https://bcgov.github.io/bcrs-shared-components/?path=/story/component-breadcrumb--home-route

And I think it looks OK in the other UIs. Why is it different in Namerequest UI? Also I'd like to find another solution that doesn't require a "special number" in various places (here, auth-web, etc).

Reply:
I dug into this a bit and it seems other UIs do indeed rely on this "special number" to set the width of the breadcrumb. It isn't immediately obvious based on the css, but you can find it in[ business-filing-ui](https://github.com/bcgov/business-filings-ui/blob/bef1b9b0d48ca308447f6205f483998ca96a7e14/src/assets/styles/layout.scss#L35), [business-edit-ui](https://github.com/bcgov/business-edit-ui/blob/ef4a03ba0fc64f7b0297fd3aa46d9aaa39eadd18/src/assets/styles/layout.scss#L35), and [business-create-ui](https://github.com/bcgov/business-create-ui/blob/d0e45f28a223a1199201a7df29e513537d30fe75/src/assets/styles/layout.scss#L37). The reason it isn't obvious is that it is just being applied to the `.container` class, but when you inspect the breadcrumb you can see that this css gets applied to it (see screenshot). I agree that I don't like using an arbitrary number. I look into a different solution for this.

![Screenshot (3)](https://github.com/bcgov/namerequest/assets/122323255/532c40ba-2c1d-4d0c-bef4-f4871bcd11c0)

Reply:
Shame on us 😆 

I accept the container width spec in the other UIs. I'd rather not see it nested in the Breadcrumb component.

Thanks for looking into this. It's not worth a ton of effort but I bet it will save some time in the future.

Reply:
Okay sounds good, so we'd like to move it back to layout.scss like in the other projects?

Reply:
Yes, but I'm assuming it's not specific to the Breadcrumbs component, right?

Reply:
Yeah that's correct, I have moved it back to layout.scss